src/exec.cpp

```diff
@@ -979,7 +982,23 @@ void Executor::raise(const std::string &exception, const std::string &info)
         ip = callstack.top();
         callstack.pop();
     }
-    fprintf(stderr, "unhandled exception %s (%s)\n", exception.c_str(), info.c_str());
+    fprintf(stderr, "unhandled exception %s (%s) at address %lu\n", exception.c_str(), info.c_str(), eip);
+    
+    if (debug != nullptr) {
+        auto line = debug->line_numbers.find(eip);
+        if (line == debug->line_numbers.end()) {
+            auto p = eip;
+            for (;;) {
+                line = debug->line_numbers.find(--p);
+                if (line != debug->line_numbers.end()) {
+                    break;
+                }
+            }
+        }
+        fprintf(stderr, "%d | %s\n", line->second, debug->source_lines.at(line->second).c_str());
+    } else {
+        fprintf(stderr, "Neon binary was not built with additional debug information so no detailed info can be displayed.\n");
```

Comment:
This message is way too long, also it's wrong. Just because no debug information is available *here*, doesn't mean the file was not built with debug information. Suggest simply `"No debug information available"` for now.